coinstacks/bitcoin/ingester/src/parseTx/index.ts

```diff
@@ -0,0 +1,55 @@
+import { BigNumber } from 'bignumber.js'
+import { Tx, Vin, Vout } from '@shapeshiftoss/blockbook'
+import { TxTransfer, Token } from '@shapeshiftoss/common-ingester'
+import { BTCParseTx } from '../types'
+
+// keep track of all individual tx components and add up the total value transferred
+const aggregateTransfer = (transfer: TxTransfer, value: string, token?: Token): TxTransfer => {
+  if (transfer) {
+    transfer.totalValue = new BigNumber(transfer.totalValue).plus(new BigNumber(value)).toString(10)
+    transfer.components.push({ value: value })
+  } else {
+    transfer = { totalValue: value, components: [{ value: value }] }
+  }
+
+  return { ...transfer, token }
+}
+
+export const parseTx = async (tx: Tx, address: string): Promise<BTCParseTx> => {
+  const pTx: BTCParseTx = {
+    ...tx,
+    address,
+    receive: {},
+    send: {},
+  }
+
+  // todo - assumptions:
+  // - all inputs spend, no change address
+  // get send amount
+  tx.vin.forEach((vin: Vin) => {
+    if (vin.isAddress === true && vin.addresses?.[0] === pTx.address) {
+      const sendValue = new BigNumber(vin.value ?? 0)
+      if (!sendValue.isNaN() && sendValue.gt(0)) {
+        pTx.send['BTC'] = aggregateTransfer(pTx.send['BTC'], vin.value || '0')
+
+        // network fee (only for sends)
+        const fees = new BigNumber(tx.fees ?? 0)
```

Comment:
Same default comment as above and the `tx.fees` check is also probably unnecessary if we want to clean that up. I can make the same update on the eth side to match